sdk/iot/core/inc/az_iot_hub_client.h

```diff
@@ -0,0 +1,505 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * @file az_iot_hub_client.h
+ *
+ * @brief definition for Azure IoT Hub device-client SDK.
+ */
+
+#ifndef _az_IOT_HUB_CLIENT_H
+#define _az_IOT_HUB_CLIENT_H
+
+#include <az_result.h>
+#include <az_span.h>
+#include <stdbool.h>
+#include <stdint.h>
+

```

Comment:
Can we order headerfile in following order:

In dir/foo.cc or dir/foo_test.cc, whose main purpose is to implement or test the stuff in dir2/foo2.h, order your includes as follows:

dir2/foo2.h.
A blank line
C system headers (more precisely: headers in angle brackets with the .h extension), e.g. <unistd.h>, <stdlib.h>.
A blank line
C++ standard library headers (without file extension), e.g. <algorithm>, <cstddef>.
A blank line
Other libraries' .h files.
Your project's .h files.
Separate each non-empty group with one blank line.


excerpt:
[https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes)

Reply:
Following AZ SDKs guidelines. I did insert a blank line per their guidance.